kubernetes/job-rake-task-staging.tmpl

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: __JOB_NAME__
+spec:
+  template:
+    spec:
+      containers:
+      - name: tove-rake-task-staging
+        image: zooniverse/tove:__IMAGE_TAG__
+        command: ["bundle",  "exec", "rake", __RAKE_TASK_NAME__]
```

Comment:
Do we want to be running `rails` command vs `rake` for Rails v5+ ?  (Not breaking, I think I read both works).

Reply:
Yeah, actually probably so. As far as I can tell, it was only `routes` and a few others were explicitly moved to `rails` and deprecated, but the docs indicate that most stuff I did with `rake` before now use `rails`, including `db:migrate`: https://guides.rubyonrails.org/6_1_release_notes.html#railties-removals

I'll update this.